cveapi/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.black]
+target-version = ["py311"]
```

Comment:
`target-version = ["py38", "py39", "py310", "py311"]` per https://github.com/psf/black/issues/751

Reply:
It also doesn't hurt to have this consistent with the rest, so I changed it.